element_electrode_localization/coordinate_framework.py

```diff
@@ -56,13 +71,124 @@ class Voxel(dj.Part):
         -> CCF.Voxel
         """
 
+    @classmethod
+    def retrieve_acronym(self, acronym):
+        """ Retrieve the DataJoint translation of the CCF acronym"""
+        return re.sub(r'(?<!^)(?=[A-Z])', '_', acronym).lower()
+
+    @classmethod
+    def voxel_query(self, x=None, y=None, z=None):
```

Comment:
If this is a classmethod, use `cls`, not `self`.  If `self` is not used, then you can make this a static method.